idd/Energy+.idd.in

```diff
@@ -39641,6 +39661,10 @@ AirTerminal:SingleDuct:VAV:NoReheat,
        \note If the field Zone Minimum Air Flow Input Method is Scheduled, then this field
        \note is optional; if a value is entered, then it is used for sizing normal-action reheat coils.
        \note If both this field and the following field are entered, the larger result is used.
+       \note The values for autosizing are picked up from the Sizing:Zone input fields 
+       \note ”Cooling Minimum Air Flow per Zone Floor Area”, ”Cooling Minimum Air Flow”, and 
```

Comment:
I haven't validated this, but these quotation marks littered through here do not look like standard plain-text quotation marks.  If they are stylized, that style should be removed and go back to regular quote marks.